app/client-app/src/components/shared/SurveyPage.jsx

```diff
@@ -168,17 +173,24 @@ const SurveyPage = ({
           </div>
 
           <Stack spacing={3} direction="row" align="center">
-            <Button
-              size="lg"
-              colorScheme="red"
-              variant="outline"
-              borderWidth="2px"
-              onClick={clearResult}
-              mr={2}
+            <Tooltip
+              label="This page requires a response before proceeding, so it cannot be cleared. You can continue to change your response before submitting."
+              isDisabled={hasOptionalComponent}
+              shouldWrapChildren
             >
-              Clear Response
-            </Button>
-            {webhookCount != null && (
+              <Button
+                size="lg"
+                colorScheme="red"
+                variant="outline"
+                borderWidth="2px"
+                onClick={clearResult}
+                mr={2}
+                disabled={!hasOptionalComponent}
```

Comment:
I think Chakra has an `isDisabled` prop for buttons? disabled is the pure HTML way; I think `isDisabled` does extra stuff like setting the HTML attribute and also adding `aria-` attributes.

The `isDisabled` Button prop in the docs: https://chakra-ui.com/docs/components/button/props